tests/infrastructure/downward-metrics.go

```diff
@@ -93,4 +99,40 @@ var _ = DescribeInfra("downwardMetrics", func() {
 		Expect(found).To(BeTrue())
 		Expect(metrics.Metrics[j].Value).To(Equal("1"))
 	})
+
+	DescribeTable("should not start a VMI if the feature is disabled", Serial, func(via libvmi.Option) {
+
+		kv := libkubevirt.GetCurrentKv(virtClient)
+		originalConfig := kv.Spec.DownwardMetrics.DeepCopy()
+
+		if originalConfig != nil {
+			By("updating the KubeVirt object to disable the downwardMetrics feature")
+			kv.Spec.DownwardMetrics = nil
+			_, err := kubevirt.Client().KubeVirt(kv.Namespace).Update(context.Background(), kv, metav1.UpdateOptions{})
+
+			Expect(err).ToNot(HaveOccurred(), "Should update kubevirt downwardMetrics configurable")
+			testsuite.EnsureKubevirtReady()
+			//TODO: additional check to ensure KV is up to date?
```

Comment:
`EnsureKubevirtReady` does this already no?

Reply:
I think so, but I've left that comment to be sure :sweat_smile: